src/pages/Item.tsx

```diff
@@ -6,34 +6,39 @@ import ProductDetail from '../components/ProductDetail';
 import CommentList from '../components/CommentList';
 import Button from '../components/Button';
 import turnBackImg from '../assets/icons/ic_turn_back.svg';
+import { ChangeEvent } from 'react';
+import { Comment } from '../types/comment';
+import { Product } from '../types/product';
 
 function Item() {
   const params = useParams();
```

Comment:
아래에서 params의 itemId 요소에 접근하고 있으므로 

```
    handleLoadItem(Number(params.itemId));
    handleLoadComments(Number(params.itemId));
```
구조분해 할당을 해서 itemId에 바로 접근할 수 있으면 더 좋을 것 같습니다!
```suggestion
  const { itemId } = useParams();

```